internal/proto/controller/api/resources/workers/v1/worker.proto

```diff
@@ -90,6 +90,9 @@ message Worker {
   // itself.
   WorkerConfig worker_config = 140 [json_name = "worker_config"];
 
+  // Output only. The number of sessions that this worker is currently handling.
+  uint32 active_session_count = 150 [json_name = "active_session_count"]; // @gotags: `class:"public"`
```

Comment:
You might want to coordinate with Jim on the number here- he's also set 150: https://github.com/hashicorp/boundary/pull/2143/files#diff-021096cfbf64554a86d2acfaeff08ec6399d84eda7954d84224c1e535380c4a8